src/vocgui/serializers.py

```diff
@@ -1,14 +1,42 @@
 from django.contrib.auth.models import Group
+from django.templatetags.static import static
 from rest_framework import serializers
 
 from .models import Discipline, TrainingSet, Document, AlternativeWord, DocumentImage
 from .utils import get_child_count
 
 
-class DisciplineSerializer(serializers.ModelSerializer):
+class FallbackIconSerializer(serializers.ModelSerializer):
+    """
+    Serializer for models with an icon field which should provide a fallback value
+    """
+
+    icon = serializers.SerializerMethodField()
+
+    def get_icon(self, obj):
+        """
+        Get the the icon if it exists and a fallback image otherwise
+
+        :param obj: The model instance
+        :type obj: ~django.db.models.Model
```

Comment:
Why ~? Typo? :-)

Reply:
In the documentation, this leads to only the last part to be the visible part of the link, while the whole module path is used to determine the correct link (see e.g. [SO](https://stackoverflow.com/questions/52395174/shorter-name-for-the-class-link-in-sphinx)).
I'm not sure if this already works in the Lunes documentation, but for Integreat, there is e.g. this example:

https://github.com/digitalfabrik/integreat-cms/blob/0da532a335b8752f8d12e0a9552c372999a92443/integreat_cms/cms/templatetags/model_tags.py#L18

renders to:

https://digitalfabrik.github.io/integreat-cms/ref/integreat_cms.cms.templatetags.html#model-tags

Reply:
Nice, didn't know that!